src/Qowaiv.Data.SqlClient/Generated/Timestamp.generated.cs

```diff
@@ -239,6 +239,8 @@ public partial struct Timestamp
     /// The <see cref="string"/> to validate.
     /// </param>
     [Pure]
+    [ExcludeFromCodeCoverage]
+    [Obsolete("Use the SVO itself instead. Will be dropped when the next major version is released.")]
```

Comment:
Could you elaborate on 'Use the SVO itself'? I would love a clear piece of code, here, or in a readme file, with which I can replace my IsValid(str) calls.

Reply:
Fair point. I Updated it to : `"Use Timestamp.TryParse(str) is { } instead"`.